aopy/postproc.py

```diff
@@ -136,6 +136,109 @@ def get_trial_targets(trials, targets):
         trial_targets[trial].append(targets[idx])
     return trial_targets
 
+def get_target_dir_from_cursor(cursor_pos, target_pos):
+    '''
+    This function calculates the instantaneous target direction from the current cursor position. Assumes angle 0 starts from the direciton of vector [1, 0]
+    This function is specific to the 2D case at the moment.
+
+    Args:
+        cursor_pos (x, y): Current cursor position in spatial coordinates
+        target_pos (x, y): Current target position in spatial coordinates
+
+    Returns:
+        A tuple containing:
+            | **relative_target_angle (float):** Absolute angle between cursor and target position [rad]
+            | **relative_target_pos (float):** Absolute position of the target relative to the cursor
+    '''
+    relative_target_pos = target_pos - cursor_pos
+    relative_target_angle = np.arctan2(relative_target_pos[1], relative_target_pos[0])
+    if relative_target_angle < 0:
+        relative_target_angle = 2*np.pi + relative_target_angle
+
+    return relative_target_angle, relative_target_pos
```

Comment:
can we just call this something like `angle_between_points(p1, p2)` to make it more general?